Sources/Sentry/Profiling/SentryProfilerDefines.h

```diff
@@ -15,7 +15,7 @@ typedef NS_ENUM(NSUInteger, SentryProfilerTruncationReason) {
     SentryProfilerTruncationReasonAppMovedToBackground,
 };
 
-static NSTimeInterval kSentryProfilerChunkExpirationInterval = 10;
+static NSTimeInterval kSentryProfilerChunkExpirationInterval = 60;
```

Comment:
`m`: It would be great to have a test failing if we change this by mistake. Then, you need to mess up the code in two different places.